web-console/src/views/query-view/column-tree/column-tree-menu/string-menu-items/string-menu-items.spec.tsx

```diff
@@ -37,4 +37,17 @@ describe('string menu', () => {
     const { container } = render(stringMenu);
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  it('matches snapshot containing remove group by button', () => {
+    const stringMenu = (
+      <StringMenuItems
+        columnName={'channel'}
+        parsedQuery={parser(`SELECT channel, count(*) as cnt FROM wikipedia GROUP BY 1`)}
```

Comment:
How is this test case different from the one above? I think you do not need to add new snapshot tests (they do not add anything as far as I can see). You need to update the existing snapshots with `jest -u`

Reply:
I've made changes to the tests per our discussion.